src/distro/distro.py

```diff
@@ -995,7 +995,7 @@ def info(self, pretty: bool = False, best: bool = False) -> InfoDict:
 
         For details, see :func:`distro.info`.
         """
-        return dict(
+        return InfoDict(
             id=self.id(),
             version=self.version(pretty, best),
             version_parts=dict(
```

Comment:
```suggestion
            version_parts=VersionDict(
```

Thanks for your patch !
On the same idea, I'd like to propose that embedded `version_parts` attribute gets also converted to `VersionDict` to silence Pylint `R1735` recommendation for src/distro/distro.py:1001:26.
WDYT ? 🙂

Reply:
Sure, I can also fix it.